redux/cardSlice.jsx

```diff
@@ -0,0 +1,35 @@
+import { createSlice, createAsyncThunk } from "@reduxjs/toolkit";
+import axios from "axios";
+
+export const fetchUsers = createAsyncThunk('fetchUsers', async () => {
```

Comment:
why do we use this ?

Reply:
ma'am we are using it get api, when fetchUsers dispatch then it calls the api and return the data.